### PR TITLE
feat: improve logging part 1

### PR DESCRIPTION
**Pull request by adschwartz**
## Description:
* service logs file name is now `<enclave-name>--<service-name>-logs.txt`
* Focus input box on Create an Enclave so you can type without clicking
* Change the logging layout so it's more compact, no borders
* Removed error messages from the logs themselves, will need to revisit how we should be displaying that.
* removed dropdown on the New Enclave button so it goes straight to the Create new Enclave modal

## Is this change user facing?
YES

### CODE REVIEW

File: enclave-manager/web/src/components/enclaves/tables/FilesTable.tsx [hunk 1]
```diff
@@ -1,53 +1,46 @@
...
         header: "Name",
-        cell: ({ row, getValue }) => (
-          <Link to={`/enclave/${enclaveShortUUID}/file/${row.original.uuid}`}>
-            <Button size={"sm"} variant={"ghost"}>
-              {getValue()}
-            </Button>
-          </Link>
-        ),
+        cell: ({ row, getValue }) =>
+          //<Link to={`/enclave/${enclave.shortenedUuid}/file/${row.original.fileUuid}`}>
```
Comment (tedim52) [inline enclave-manager/web/src/components/enclaves/tables/FilesTable.tsx]: can we remove dead code?

**Review by tedim52 — APPROVED** [review]

File: enclave-manager/web/src/components/enclaves/tables/FilesTable.tsx (hunk 1 above)
Comment (adschwartz) [reply enclave-manager/web/src/components/enclaves/tables/FilesTable.tsx]: yeah we should. Not sure if Ben is still working on this. He might be.

File: enclave-manager/web/src/emui/enclaves/enclave/service/logs/ServiceLogs.tsx [hunk 2]
```diff
@@ -45,7 +45,8 @@ export const ServiceLogs = ({ enclave, service }: ServiceLogsProps) => {
         if (cancelled) {
           return;
         }
-        setLogLines((logLines) => [...logLines, { message: `Error: ${stringifyError(error)}`, status: "error" }]);
+        // TODO: We need a way to show this error
+        alert (`Error: ${stringifyError(error)}`)
```
Comment (Dartoxian) [inline enclave-manager/web/src/emui/enclaves/enclave/service/logs/ServiceLogs.tsx:49]: Easiest option that's nicer than `alert` would be to use a `toast`. Slightly nicer would be to store the message in state called `error` and use a `KurtosisAlert` component under/above the log viewer to display the error.

**Review by Dartoxian — APPROVED** [review]

File: enclave-manager/web/src/emui/enclaves/enclave/service/logs/ServiceLogs.tsx (hunk 2 above)
Comment (adschwartz) [reply enclave-manager/web/src/emui/enclaves/enclave/service/logs/ServiceLogs.tsx:49]: Agreed and I thought about the toast but we don't actually want a toast. We need to auto-reconnect and not bother the user. As mentioned in the PR text we need to do more work here. It's just temporary to surface the error.

**Pull request merged**